src/plugins/kibana/server/routes/api/ingest/register_processors.js

```diff
@@ -1,8 +1,22 @@
 import _ from 'lodash';
 import handleESError from '../../../lib/handle_es_error';
-import handleResponse from '../../../lib/process_es_ingest_processors_response';
 import { keysToCamelCaseShallow, keysToSnakeCaseShallow } from '../../../../common/lib/case_conversion';
 
+export function handleResponse(response) {
```

Comment:
This could also maybe be considered a converter, but since there can't really be a matching kibanaToES version, it probably doesn't make sense to split it out.